javascripts/tagmanager.js

```diff
@@ -1635,6 +1635,17 @@
                 throwError: throwError,
                 Container: Container,
                 addContainer: function (containerConfig, templates) {
+                    var mtmSetDebugFlag = urlHelper.getQueryParameter('mtmSetDebugFlag');
+                    if (mtmSetDebugFlag) {
+                        if (mtmSetDebugFlag == 1) {
+                            var date = new Date();
+                            date.setTime(date.getTime() + (7 * 24 * 60 * 60 * 1000));
+                            document.cookie = 'mtmPreviewMode=mtmPreview' + containerConfig.idsite + '_' + containerConfig.id + '%3D1;expires=' + date.toUTCString() + ';SameSite=Lax';
```

Comment:
I wonder if it's maybe better in both cases the `encodeURIComponent()` the `idsite` and `container config ID`. I know they only have specific characters in there just wondering in case there's somehow an issue that the containerId could be changed might be good to have